app/src/main/java/com/duckduckgo/app/httpsupgrade/db/HttpsUpgradeDomain.kt

```diff
@@ -21,13 +21,4 @@ import android.arch.persistence.room.Ignore
 import android.arch.persistence.room.PrimaryKey
 
 @Entity(tableName = "https_upgrade_domain")
-class HttpsUpgradeDomain() {
```

Comment:
The `android.arch.persistence.room.Ignore` import is now unused. A few others have sneaked in too:
<img width="387" alt="imports" src="https://user-images.githubusercontent.com/1652407/34789269-8cc800e0-f635-11e7-8809-81d1b2d161fa.png">